src/summary.rs

```diff
@@ -120,70 +120,98 @@ impl Summary {
 impl Message {
     /// Returns a summary text.
     async fn get_summary_text(&self, context: &Context) -> String {
-        let mut append_text = true;
-        let prefix = match self.viewtype {
-            Viewtype::Image => stock_str::image(context).await,
-            Viewtype::Gif => stock_str::gif(context).await,
-            Viewtype::Sticker => stock_str::sticker(context).await,
-            Viewtype::Video => stock_str::video(context).await,
-            Viewtype::Voice => stock_str::voice_message(context).await,
-            Viewtype::Audio | Viewtype::File => {
+        let (emoji, type_name, type_file, append_text) = match self.viewtype {
```

Comment:
Probably it would be easier to read if it was
```
let (emoji, type_name, type_file, append_text);
match self.viewtype {
    <type> => {
        emoji = ...;
        ...
    },
    ...
}
```

Reply:
indeed, that is better to read. i added a commit, thanks a lot!